src/components/result-card/ResultCard.tsx

```diff
@@ -0,0 +1,57 @@
+import { useGame } from "../../hooks";
+
+const ResultCard = ({
+  question,
+  options,
+  questionIndex,
+  correctOption,
+}: {
+  question: string;
+  options: string[];
+  questionIndex: number;
+  correctOption: number;
+}) => {
```

Comment:
suggestion: similar to context types, create a component types folder and have all the component types there.

Reply:
ok will refactor in the next pr